Makefile

```diff
@@ -22,6 +22,7 @@ K_LIB := $(K_RELEASE)/lib/kframework
 export K_RELEASE
 
 LIBRARY_PATH       := $(LOCAL_LIB)
+K_INCLUDE_PATH     := $(BUILD_LOCAL)/include/k
```

Comment:
Is there a technical reason to have this at `include/k`? I think we should just have it at `include`, because then people can just do `requires "krypto.md"` from a global install, instead of having to do `requires "k/krypto.md"`, if they have `-I /usr/bin/include/kframework` set.

Though, our global install of K puts files at `/usr/bin/include/kframework`, so maybe we should install to `$(BUILD_LOCAL)/include/kframework/blockchain-k-plugin/krypto.md`, then do `K_INCLUDE_PATH=$(BUILD_LOCAL)/include/kframework`, and do `requires "blockchain-k-plugin/krypto.md"` in our files. This seems most consistent with (i) how a global K install will work, and (ii) some concept of separate "packages" you can install with different K includes in each.

@dwightguth thoughts? I would like to use this to setup some notion of global packages, with the hope of one day, for example, having the Firefly build able to just depend on a globally installed KEVM instead of a submodule.

Reply:
Logically, if you want to install a k package globally, you would want to install it to `/usr/local/include/kframework`. It's fairly normal for includes to use directory names as namespaces, so presumably you would install and require under a subdirectory. Are you sure you don't have a typo in your comment, though? The path `/usr/bin/include` seems incredibly sketchy to me.

Reply:
I updated the `requires` in the source files to have the form: `requires "blockchain-k-plugin/krypto.md"` as mentioned above.

I also updated the makefile for the blockchain plugin to install inside the `blockchain-k-plugin` subdir at the installation prefix.